Fix #16804 - table CSS for number alignment

lrb2 opened the pull request:
### Description

The tables generated by the `makeGrid` JS function used the `pma_table` CSS class instead of `pma-table` as used throughout the CSS styles, so many CSS rules were not being applied to these tables. In addition, the only two `pma_table` rules in each theme were also changed to `pma-table`.

Fixes #16804

williamdes reviewed (approved):
very nice fix !

williamdes commented on themes/bootstrap/scss/_common.scss:
```diff
@@ -2374,7 +2374,7 @@ input#auto_increment_opt {
 }
 
-.pma_table {
+.pma-table {
```
I am not sure this is a wanted side effect of this PR but it breaks css styles see the block above the table is wrong

williamdes replied:
![image](https://user-images.githubusercontent.com/7784660/128616865-48a34fa8-69ac-44f3-9fd8-1b4aa8debeba.png)

lrb2 replied:
Ah, I looked right over that and didn't realize that those styles didn't need to be applied to the other `pma-table`'s, just the ones that had been `pma_table`'s. I just updated my commit to fix that!

mauriciofauth commented on js/src/makegrid.js:
```diff
@@ -2190,7 +2190,7 @@ var makeGrid = function (t, enableResize, enableReorder, enableVisib, enableGrid
 
     // add table class
-    $(t).addClass('pma_table');
+    $(t).addClass('pma-table');
```
The `.pma-table` class is not the same as the `.pma_table` class. `.pma-table` is the style that was applied to the `table` HTML tag and was extracted to a CSS class. The `pma-` prefix was added to avoid conflict with the Bootstrap's `.table` class.

lrb2 committed:
Fix text alignment in some Bootstrap tables:
Replaced instances of the `right` CSS class (used only for `pma-table`s)
with Bootstrap's `text-right`

Signed-off-by: Lukas Bixen <lrb2_@outlook.com>

lrb2 commented:
Thanks for your comment @mauriciofauth! I found the solution that a few tables that use Bootstrap were still using the old `right` class instead of Bootstrap's `text-right`. I do think that it would be nice if the `pma_table` class was renamed to `pma-grid` (since it's made using the `makeGrid` function) or something because it's easy to confuse `pma-table` and `pma_table` right now. But if you're used to it then changing it might cause more harm than good.

williamdes commented:
For reference this was made on master and seems similar: d3c31fabd4012cd483ff268068d2399c36fb5b9f
